test/statsCases/modules-assets/expected.txt

```diff
@@ -0,0 +1,16 @@
+Hash: e6165e79d5ae833a8859
+Time: Xms
+Built at: Thu Jan 01 1970 00:00:00 GMT
+                               Asset       Size  Chunks             Chunk Names
+155613d841c408f2a34b9c81cdfd3e8c.png   37.7 KiB          [emitted]  
+                                0.js  607 bytes       0  [emitted]  
+                             main.js   6.76 KiB       1  [emitted]  main
+Entrypoint main = main.js
+chunk    {0} 0.js 99 bytes <{1}> [rendered]
+    [0] ./node_modules/a/1.png 82 bytes {0} [built]
+    [1] ./node_modules/a/index.js 17 bytes {0} [built]
+chunk    {1} main.js (main) 12 bytes >{0}< [entry] [rendered]
+    [2] ./index.js 12 bytes {1} [built]
+   [0] ./node_modules/a/1.png 82 bytes {0} [built]
```

Comment:
The `assets` info isn't in the stats string output. It's only in the stats json.

Maybe you can expand this PR a bit and add support for showing emitted files in the Stats.

I could image a small flag like this:

``` text
[0] ./node_modules/a/1.png 82 bytes {0} [built] [1 asset]
```

You also need include a flag `moduleAssets` to show/hide this information. It should be enabled by default in json output but disabled by default for toString, so it's backward-compatible.

Reply:
@sokra 
Sure, I will try to work on int.